smf/ntp/manifest/manifest.xml

```diff
@@ -41,10 +41,10 @@
 
   <dependency name='chrony-setup' grouping='require_all' restart_on='refresh'
     type='service'>
-  <service_fmri value='svc:/oxide/zone-network-setup:default' />
```

Comment:
Why is this dependency being removed? AFAIK all omicron services need to depend on the zone-network-setup service